openpype/modules/ftrack/event_handlers_user/action_create_folders.py

```diff
@@ -196,6 +228,27 @@ def launch(self, session, entities, event):
             if not os.path.exists(path):
                 os.makedirs(path)
 
+        self.log.info("Creating local template folders...")
+
+        if from_local:
+            try:
+                # Get paths based on presets
+                basic_paths = get_project_basic_paths(project_name)
+                if not basic_paths:
+                    pass
+                # Invoking OpenPype API to create the project folders
+                create_project_folders(
+                    basic_paths,
+                    project_name,
+                    root_paths=[work_root]
+                )
+                self.log.info("successfully created local template folders.")
```

Comment:
Moved log and fix condition.
```suggestion
        if from_local:
            self.log.info("Creating local template folders...")
            try:
                # Get paths based on presets
                basic_paths = get_project_basic_paths(project_name)
                if basic_paths:
                    # Create the project folders
                    create_project_folders(
                        basic_paths,
                        project_name,
                        root_paths=[work_root]
                    )
                    self.log.info("successfully created local template folders.")
```